stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -162,11 +181,17 @@ end
 similar(A::Union{Symmetric,Hermitian}, ::Type{T}, dims::Dims{N}) where {T,N} = similar(parent(A), T, dims)
 
 # Conversion
-Matrix(A::Symmetric) = copytri!(convert(Matrix, copy(A.data)), A.uplo)
+function Matrix(A::Symmetric)
+    B = copytri!(convert(Matrix, copy(A.data)), A.uplo)
```

Comment:
Can `convert(Matrix, copy(A.data))` be simplified to `Matrix(A.data)`?

Reply:
That sounds plausible, but it was like that before, so I don't know what considerations led to the `copy`+`convert` construct.